.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,18 +4,20 @@ about: Is something not right? Let us know.
 
 ---
 
-  - Did you find a bug in datatable, or maybe the bug found you?
-    Tell us what it is.
+- Did you find a bug in datatable, or maybe the bug found you?
+  Tell us what it is.
 
-  - How to reproduce the bug?
-    **This part is the most important**. Please provide a code snippet which, when executed in console,
-    demonstrates the problem. If necessary, attach any data files to the issue.
+- How to reproduce the bug?
+  **This part is the most important**. Please provide a code snippet which, when
+  executed in console, demonstrates the problem. If necessary, attach any data
+  files to the issue.
 
-  - What was the expected behavior?
-    In case it is not obvious, please tell us what result should your code produce.
+- What was the expected behavior?
+  In case it is not obvious, please tell us what result should your code
+  produce.
 
-  - Your environment?
-    What is your datatable version, python version, and operating system?
+- Your environment?
+  What is your datatable version, python version, and operating system?
 
-  - Tag the issue with `[bug]` or `[segfault]` (depending on whether it crashes Python or not)
-    Thank you for contributing, and sorry for any inconvenience...
+- Tag the issue with `[bug]` or `[segfault]` (depending on whether it crashes
+  Python or not). Thank you for contributing, and sorry for the inconvenience...
```

Comment:
>Thank you for contributing, and sorry for the inconvenience...
Should that line be part of the "tag the issue" bullet? It seems like it should be outside of the steps. Also, you can remove the ellipses and just end with a period.

Reply:
I moved that sentence into a new line.